src/Wrapper.js

```diff
@@ -39,8 +39,7 @@ const propTypes = {
     PropTypes.string,
   ]),
   value: PropTypes.oneOfType([
-    PropTypes.bool,
-    PropTypes.string,
+    PropTypes.any,
   ]),
```

Comment:
I think this can just be `value: PropTypes.any,` now that there's only 1.

Reply:
Ha, yes!